README.md

```diff
@@ -70,6 +71,7 @@ FloatingButton(mainButtonView: mainButton, buttons: buttons, isOpen: $isOpen)
 `startAngle`  
 `endAngle`  
 `radius` - distance between center of main button and centers of submenu buttons  
+`rotation` - changes the button layout direction from the startAngle to the endAngle
```

Comment:
I think 'rotation' is reserved for another use case. Since it's a button layout direction - would you please call it 'layoutDirection'?

Reply:
Good idea!